Automattic/jetpack
# Add jetpack-mu-wpcom plugin skeleton

#### Changes proposed in this Pull Request:

Adds the `jetpack-mu-wpcom` plugin skeleton.

#### Other information:


#### Jetpack product discussion

Asana: 1203453160997315-as-1203460747328995/f
PT: p9dueE-6jY-p2

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

Plugin doesn't do anything yet; proofread to make sure the structure looks to be in place.

Mirror repo created: https://github.com/Automattic/jetpack-mu-wpcom
